### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by mjpost** [acl-org/acl-anthology#551]
It'd be nice to have a logo for the Anthology. A variation on the ACL block "a", but distinguishable. One idea is to incorporate that "a" with the coarse shape of a two-column paper.

**Comment by xavierrollet** [comment]
Hi everyone, this is the suggestion of a logo that i made, feedbacks are welcome :-)

Kind regards,

![Anthologylogo](https://user-images.githubusercontent.com/56123802/66484922-2e145300-eaa8-11e9-8066-6674395e69d3.png)

**Comment by esalesky** [comment]
Here is a take on a possible logo using the "a" with a scroll, with @pocaguirre

![acl_anthology_logo](https://user-images.githubusercontent.com/4117932/85348285-21fee300-b4c9-11ea-86c3-2133d64dafa6.png)

**Comment by mjpost** [comment]
Thanks for the contribution! (also to @xavierrollet, who I never responded to)

I personally really like the simplicity and plainness of the current ACL logo. My favorite variant of it cleaves it from top to bottom, making clear that the "a" is formed from a block "c" and "l". It seems like something could be done with these two halves, analogous to an opened book (or two-column paper).

![acl_logo_split](https://user-images.githubusercontent.com/455256/85354020-98a2dd00-b4d7-11ea-955a-c51004ef77da.png)

**Comment by mbollmann** [comment]
Yes, thanks for the suggestions! I like both the abstractness of the first one and the scroll idea of the second one.

A litmus test for any logo, I believe, is that it should work well in smaller sizes. Not only in the header of the ACL Anthology website (where we could play around with the design if we had a logo that's better shown a bit larger) but also for the Twitter account etc. – on my device, that means around ~50px in width.

Here's what the current suggestions look like at that size, just to give you an idea:

![acla1-s](https://user-images.githubusercontent.com/11348502/85392411-c45faa80-b54b-11ea-8cf8-a40b4866682b.png)
![acla2-s](https://user-images.githubusercontent.com/11348502/85392416-c75a9b00-b54b-11ea-89a8-50c704700aeb.png)

**Comment by esalesky** [comment]
Good points! What about something like these, kind of in-between? 
In the second, could also move the column break for the red text over to show 'C' and 'L' more. 
   
![anthology-logo-papers-mini](https://user-images.githubusercontent.com/4117932/85451501-f41cab80-b567-11ea-8542-38ecfafe3543.png) ![anthology-logo-textonly-mini](https://user-images.githubusercontent.com/4117932/85451500-f3841500-b567-11ea-8957-cc66740912d9.png)
  
![anthology-logo-papers](https://user-images.githubusercontent.com/4117932/85451483-ef57f780-b567-11ea-982d-8978bd76a836.png)
![anthology-logo-textonly](https://user-images.githubusercontent.com/4117932/85451482-ef57f780-b567-11ea-9e66-40db36c54e36.png)
 
(Thanks, this is fun!)

**Comment by mjpost** [comment]
This is getting somewhere! What about a version that is more of a square (i.e., move the "a" up, and clip the bottom up higher). Squares fit more generally across media.

I think moving the column break would help accentuate the logo but might look strange and is probably unnecessary, maybe worth seeing though.

**Comment by dowobeha** [comment]
> Thanks for the contribution! (also to @xavierrollet, who I never responded to)
> 
> I personally really like the simplicity and plainness of the current ACL logo. My favorite variant of it cleaves it from top to bottom, making clear that the "a" is formed from a block "c" and "l". It seems like something could be done with these two halves, analogous to an opened book (or two-column paper).
> 
> ![acl_logo_split](https://user-images.githubusercontent.com/455256/85354020-98a2dd00-b4d7-11ea-955a-c51004ef77da.png)

I really like this.

**Comment by esalesky** [comment]
Squared with a subtle "c" and "l" separation line: 

![anthology-logo-papers-square-mini](https://user-images.githubusercontent.com/4117932/85475368-fb51b280-b583-11ea-8443-9d102fec292a.png)
![anthology-logo-papers-square](https://user-images.githubusercontent.com/4117932/85475370-fb51b280-b583-11ea-9858-c53c7eac80fc.png)

**Comment by knmnyn** [comment]
Here's my submission taking some of your suggestions into consideration.
I used a famous NLP text that probably most of us know.
![image](https://user-images.githubusercontent.com/217067/85504637-ff7ede00-b61e-11ea-8cbf-cd7dca25c7f1.png)

![image](https://user-images.githubusercontent.com/217067/85504670-0e659080-b61f-11ea-8d2b-1426c88e2c45.png)

**Comment by mjpost** [comment]
Min, I really like this one! It keeps the design while adding something new, and I think would fit in places (e.g., the Anthology landing page, upper left corner) where the scroll version wouldn't fit as well.

I wonder if there is some quintessential or central Anthology paper that would serve well for the text. Something from COLING 1965? [Syntax et Interpretation](https://www.aclweb.org/anthology/C65-1030/)?

(I also suggest narrowing the gap between the "c" and "l").

**Comment by mjpost** [comment]
Min, I really like this one! It keeps the design while adding something new, and I think would fit in places (e.g., the Anthology landing page, upper left corner) where the scroll version wouldn't fit as well.

I wonder if there is some quintessential or central Anthology paper that would serve well for the text. Something from COLING 1965? [Syntaxe et Interpretation](https://www.aclweb.org/anthology/C65-1030/), perhaps?

**Comment by knmnyn** [comment]
Sure. I took this text because it is the first text from the Penn Treebank
and many #NLProc know it.

Please suggest another corpus to use. I will just sub in the text. By right
it would be even better if it was text from the publication that introduced
it.  E.g., https://repository.upenn.edu/cis_reports/237/

I've attached the PPTX that I used to make the logo, in case someone else
wants to have a go at this.  To make the text dense enough to show up
nicely, I justified and manually added spaces to make the "kerning" between
words smaller.  I chose Times New Roman as this is the typical font found
in Anthology pubs.

- Min

**Comment by mjpost** [comment]
This text is definitely recognizable! My only thought is that technically it's LDC, not Anthology. The Treebank paper could be a good one (I'm surprised you didn't like to the Anthology version!) Anyway, this is just nitpicking. I'm not sure what the right answer is here.

BTW, the attachment didn't come through to Github over email. Perhaps you have to post it directly.

**Comment by mbollmann** [comment]
My thoughts on text in the logo: Will this ever actually be readable at the scales that people will see the logo? I like the gimmicky aspect of it, but feel that from a design perspective, squiggles like in @esalesky's version (maybe a bit larger though) might look better when scaled down to icon size.

**Comment by mjpost** [comment]
I agree a caricature of text is preferable in a logo. It could be a nice easter egg, however, if zooming in on a vector version of the graphic revealed some central NLP text. But I don't think it's necessary.

**Comment by esalesky** [comment]
(a fun fact on the scribbles -- they are actually text in the "Redacted Script" font, which is cool because they then vary naturally in length etc. it's probably too sneaky to be a real easter egg, but any text could be used -- I just copy pasted "The ACL Anthology currently hosts 57090 papers on the study of computational linguistics and natural language processing" many times)

**Comment by mbollmann** [comment]
Idea: Can we make the logo consist _only_ of scribbles (or text), without the bounding boxes from @knmnyn's version? Kind of like in @esalesky's [second version here](https://user-images.githubusercontent.com/4117932/85451482-ef57f780-b567-11ea-9e66-40db36c54e36.png), but just the logo part, and with a lot fewer but larger lines so it doesn't look so "faded-out" when scaled down to icon size. This could work with or without a paper/scroll framing it.

I wanted to try sketching this myself, but don't have time right now, so if someone else wants to get to it first ...? :smile:

**Comment by esalesky** [comment]
Tried this and ended up inverting it so the text is white on a red background; with bigger text, there's more white space, so the ACL outline was less obvious. 
Could use in different places with and without the papers framing it, as makes sense. 

I've also [posted](https://drive.google.com/file/d/1BjDsv3ZbMzIKHh_UJKWx9nr4Ilq0bl59/view?usp=sharing) my powerpoint on drive in case someone else wants to fiddle with it directly, though you'll need [this font](https://www.1001fonts.com/redacted-font.html). 

![logo2](https://user-images.githubusercontent.com/4117932/85957050-fa3fcd00-b957-11ea-988f-2d929ecdbbe3.png)
![logo](https://user-images.githubusercontent.com/4117932/85957048-f744dc80-b957-11ea-9db9-c27ab22ec92b.png)

**Comment by knmnyn** [comment]
Hi @mjpost , all: I couldn't add the attachment in Github on the cmdline or here in the web browser, but am putting a link to a Google Drive hosting of the pptx.

https://drive.google.com/file/d/10Jd4YO3lPobElinZYTd0pgozQTH6Tuv1/view?usp=sharing
